src/app/SIPUserAgents/SIPNotifierClient.cs

```diff
@@ -51,7 +51,7 @@ public class SIPNotifierClient
         private string m_authDomain;
         private string m_authPassword;
         private string m_filter;
-        private int m_expiry;
+        private long m_expiry;
         private int m_localCSeq;

```

Comment:
What's the reason to make this long and not UInt32?

Reply:
Just kept it uniform with the rest of the project.
long in SIPHeader, long there, long in SIPRegistrationClient